src/strategy/__tests__/redis_cache_provider.test.ts

```diff
@@ -175,35 +184,35 @@ describe("getExtendedRedisCacheProvider#save", () => {
       callback(null, JSON.stringify(invalidCachedRequestData))
     );
     const redisCacheProvider = getExtendedRedisCacheProvider(mockRedisClient);
-    const cacheSAMLResponse = await redisCacheProvider
-      .get(expectedRequestID)
-      .run();
+    const cacheSAMLResponse = await redisCacheProvider.get(expectedRequestID)();
     expect(mockGet.mock.calls[0][0]).toBe(`SAML-EXT-${expectedRequestID}`);
     expect(isRight(cacheSAMLResponse)).toBeFalsy();
-    expect(cacheSAMLResponse.value).toEqual(expect.any(Error));
+    if (isRight(cacheSAMLResponse)) {
```

Comment:
the following `expect` here is never executed since the preceeding line states that `isRight` is false
(you probably want `isLeft` here)

Reply:
You are right! Fixed 😃